src/plugins/data/common/index_patterns/index_patterns/index_pattern.ts

```diff
@@ -192,11 +202,13 @@ export class DataView implements IIndexPattern {
       };
     });
 
+    const runtimeFields = this.getRuntimeMappings();
+
     return {
       storedFields: ['*'],
       scriptFields,
       docvalueFields,
-      runtimeFields: this.runtimeFieldMap,
```

Comment:
@mattkime it is usually better not to expose a (mutable) class private ref object to consumers as potentially they could do anything with it.